topics/cpp/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-aliases: c-plus-plus, cpp11, cpp14, cpp17, cpp98, cpps, cplusplus, cplusplus-11
+aliases: cplusplus, c-plus-plus, cpps, cpp98, cpp03, cpp11, cpp14, cpp17, cpp20, cpp0x, cpp1y, cpp1z, cpp2a, cplusplus-11
```

Comment:
These topics don't have any public repos associated with them. What do you think about removing them?
```suggestion
aliases: cplusplus, c-plus-plus, cpps, cpp98, cpp03, cpp11, cpp14, cpp17, cpp20, cpp1z, cplusplus-11
```

Reply:
Since cpp1z (aka cpp17) has a public repo, I decided to add these other alternate version names as well. They might seem pointless right now, but someone might use these aliases in the future. Is it actually a problem if they're "dead" aliases?

Reply:
OK thanks for the explanation. I think it’s fine to keep these then.